src/Reflection/ReflectionEnumCase.php

```diff
@@ -86,6 +86,16 @@ public function getName(): string
         return $this->name;
     }
 
+    /** @throws LogicException */
+    public function getValueExpression(): Node\Expr
+    {
+        if ($this->value === null) {
+            throw new LogicException('This enum case does not have a value');
+        }
```

Comment:
Hmm, this API is a bit "meh".

How does a consumer prevent calling this upfront, if no value exists?

Reply:
I did the same as https://github.com/Roave/BetterReflection/blob/6.0.x/src/Reflection/ReflectionEnumCase.php#L100

Consumer should probably check `ReflectionEnum::isBacked()`

Reply:
Let's add that to the docblock as documentation 👍

Reply:
Added.